easybuild/easyconfigs/r/R/R-4.0.0-foss-2020a.eb

```diff
@@ -43,6 +43,7 @@ dependencies = [
     ('GSL', '2.6'),  # for RcppGSL
     ('ImageMagick', '7.0.10-1'),  # for animation
     ('GLPK', '4.65'),  # for Rglpk
+    ('GDAL', '3.0.4', '-Python-3.8.2'),  # for sf
```

Comment:
Hmm, looks like this isn't a good idea, because it introduces conflicts between Python 2 & 3 for stuff that depends on R.

I'll flesh out the `sf` extension to a separate easyconfig.